### PR TITLE
Jasonlistingfilter

### PR DESCRIPTION
**Pull request by jason-ngan**

### CODE REVIEW

File: AirbnbApplication/src/main/webapp/search.jsp [hunk 1]
```diff
@@ -33,6 +33,11 @@
 	            <label for="hostName">Host Name:</label>
 	            <input type="text" name="hostName" class="form-control" value="${fn:escapeXml(param.hostName)}">
 	        </div>
+	        
+	        <div class="form-group">
+	            <label for="listingName">Host Name:</label>
```
Comment (kabraambika) [inline AirbnbApplication/src/main/webapp/search.jsp:38]: Why there is listingname field in host table ? @jason-ngan , We need to remove this

Comment (jason-ngan) [reply AirbnbApplication/src/main/webapp/search.jsp:38]: That was supposed to be Listing Name, I think I was implementing listingname but started doing propertylocation instead lol

Comment (kabraambika) [reply AirbnbApplication/src/main/webapp/search.jsp:38]: Okay, then do you want to complete listing name first ? then i can merge in main

**Review by kabraambika — APPROVED** [review]
Good to go!

**Pull request merged**